meteor-app/imports/ui/assessment/assessment-add-review.js

```diff
@@ -4,15 +4,25 @@ import { Button, Icon, Image, Header, Segment } from 'semantic-ui-react'
 
 const AssessmentAddReview = (props) => {
   const stepsToDisplay = props.steps.filter(step => {
-       if(step.stepTitle != "Review" && step.stepTitle != "Customer Details") {
-         return step
-       }    
+    if(step.stepTitle != "Review" && step.stepTitle != "Customer Details") {
+      return step
+    }    
   })
   return (
     <Segment style={{ textAlign: 'left' }}>
       <Header as='h1' content='Review your details:' textAlign='center' />
+      <Segment padded='very' >
+        <h2>
+          Total Price
+        </h2>
+        <div>
+        <div>{props.formData.serviceCost ? "Total Service Cost: $" + props.formData.serviceCost/100 : ''}</div>
```

Comment:
This code is kind of messy. I see what you are trying to do, ie not show the line if that item is zero. I would leave them all showing, it's fine in an invoice to show parts cost of zero. It also means the layout is always the same, which is better for printing anyway, and it also allows items to be written on the paper form (which could happen :))